src/test/ui/span/issue-24690.stderr

```diff
@@ -4,6 +4,7 @@ error: unused variable: `theOtherTwo`
 20 |     let theOtherTwo = 2;
    |         ^^^^^^^^^^^
    |
+   = note: #[deny(warnings)] implies #[deny(unused_variables)]
```

Comment:
it feels like this should be attached to the message below (`lint level defined here`), and also that it should only appear when that message appears (meaning only once), what do you think?